<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardGui</class>
 <widget class="QWidget" name="DashboardGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_sidebar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>184</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_dashboard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>184</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_contacts">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>184</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;</string>
    </property>
    <property name="text">
     <string>Contacts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Baskerville&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>184</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(220,20,60);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resources">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>184</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;</string>
    </property>
    <property name="text">
     <string>Resources</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>60</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_main">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>0</y>
     <width>648</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QScrollArea" name="scrollArea_dashboard">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>49</y>
       <width>571</width>
       <height>521</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_dashboard">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>569</width>
        <height>519</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
<!--<<<<<<< HEAD
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollArea_dashboard">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>477</width>
           <height>447</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
=======-->
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
<!-->>>>>>> d56ad766ba4c21c671197afec5d92e212ef59afb-->
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
